iml-agent/src/poller.rs

```diff
@@ -100,6 +101,7 @@ pub async fn create_poller(
                 now,
             );
 
+            #[allow(unused_must_use)]
             tokio::spawn(async move {
                 fut.await.map_err(|e| {
```

Comment:
```rust
if let Err(e) = fut.await {
    error!("{}", e);
}
```

This will allow for the error to be "consumed" (even though it already is since the `map_err` returns unit, and we can avoid adding an annotation.

Reply:
Done